system/Commands/Translation/LocalizationFinder.php

```diff
@@ -155,14 +155,28 @@ private function process(string $currentDir, string $currentLocale): void
 
         $this->writeIsVerbose('Files found: ' . $countFiles);
         $this->writeIsVerbose('New translates found: ' . $countNewKeys);
+        $this->writeIsVerbose('Bad translates found: ' . count($badLanguageKeys));
+
+        if ($this->verbose && $badLanguageKeys !== []) {
+            $tableBadRows = [];
+
+            foreach ($badLanguageKeys as $key => $value) {
+                $tableBadRows[] = [$key, $value];
+            }
+
+            CLI::table($tableBadRows, ['№', 'Bad key']);
```

Comment:
What do you need the No. for? I want the file paths.

```suggestion
            CLI::table($tableBadRows, ['№', 'Bad Key']);
```

Reply:
The number is just for understanding which line. I think file paths are not an option to show. Since one language key can be in ~1000 different files. How do you see it?

Reply:
If a bad key is used 1000 times, then those 1000 locations should be shown.


| No. | Bad Key | File |
| ----|-------- | ------------- |
| 1  | BadKey1  | path/to/file1.php |
| 2  | BadKey1  | path/to/file2.php |

Reply:
Heh, I'll try to do it.